.github/workflows/release.yml

```diff
@@ -0,0 +1,263 @@
+name: Release
+
+on:
+  # push:
+  #   branches: [auto-release-workflows]
+  workflow_dispatch:
+
+jobs:
+    
+  release:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      R_REMOTES_NO_ERRORS_FROM_WARNINGS: true
+      RSPM: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"
+
+    steps:
+      - name: Setup Git repository
+        uses: actions/checkout@v4
+
+      - name: Update Ubuntu packages
+        run: sudo apt-get update
+        
+      - name: Setup R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: 'release'
+
+      - name: Setup pandoc
+        uses: r-lib/actions/setup-pandoc@v2
+        with:
+            pandoc-version: '2.14.0.3'
+
+      - name: Install rmarkdown
+        run: install.packages("rmarkdown")
+        shell: Rscript {0}
+
+# Get previous version and then create the next version   
+      - name: Application Version
+        id: version
+        uses: paulhatch/semantic-version@v5.0.2
+        with:
+          tag_prefix: "v"
+          version_format: "v${major}.${minor}.${patch}"
+
+# Create version.txt to use when changing the version in the index.md
+      - name: Create version.txt
+        run: echo ${{ steps.version.outputs.version }} > version.txt
+
+# Update version and date in SS330_User_Manual.tex
+      - name: Update version and date in manual
+        run: |
+          version <- readLines("version.txt", encoding = "UTF-8")
+          version <- gsub("v", "", version)
```

Comment:
imo it is better to combine these two lines into a single call rather than create something and then augment it but not rename it. For example you could write
```
version <- readLines("version.txt", encoding = "UTF-8") |>
  gsub("v", "", version)
```
or
```
version <- gsub("v", "", readLines("version.txt", encoding = "UTF-8"))
```
or
```
version_raw_text <- readLines("version.txt", encoding = "UTF-8")
version <- gsub("v", "", version_raw_text)
```
None of this really matters because above I suggest a way to get rid of the preceding "v" which would remove the `gsub()` call anyway. I am just giving these examples to show a different way of writing the same code that imo leads to less future bugs when someone tries to augment the code but they weren't the ones that wrote it.